packages/components/ng-translate-async-loader/src/index.js

```diff
@@ -4,6 +4,15 @@ import factory from './factory';
 
 const moduleName = 'ngTranslateAsyncLoader';
 
-angular.module(moduleName, []).factory('asyncLoader', factory);
+angular
+  .module(moduleName, [])
+  .factory('asyncLoader', factory)
+  .run(
```

Comment:
Both `$transitions` and `$translate` come from `@uirouter/angularjs` and `angular-translate` which are not defined as peerDependencies of this module.

In some way, it looks like a Breaking Change but out of curiosity, shouldn't be done inside `@ovh-ux/manager-core`?

Reply:
i think you're right it would make sense to add this in core, i'm going to update this PR.